src/reuse/helper/errorHandler.ts

```diff
@@ -53,8 +55,8 @@ export default class ErrorHandler implements IErrorHandler {
 
   // =================================== HELPER ===================================
 
-  private _retrieveFunctionNameFromStack(errorObject: Error): string {
-    if (errorObject.stack) {
+  private _retrieveFunctionNameFromStack(errorObject: unknown | Error): string {
+    if (errorObject instanceof Error && errorObject.stack) {
```

Comment:
as discussed in our call, function name should be extracted from lowest level of the call stack meaning the initial function the use has called. But you can do it in a separate PR.